Source/Scene/GlobeTranslucencyFramebuffer.js

```diff
@@ -39,109 +30,72 @@ function GlobeTranslucencyFramebuffer() {
 Object.defineProperties(GlobeTranslucencyFramebuffer.prototype, {
   classificationTexture: {
     get: function () {
-      return this._colorTexture;
+      return this._framebuffer.getColorTexture();
     },
   },
   classificationFramebuffer: {
     get: function () {
-      return this._framebuffer;
+      return this._framebuffer.framebuffer;
+    },
+  },
+  packedDepthFramebuffer: {
```

Comment:
A bunch of these getters were added for easy access in specs. It was a tradeoff of making this file longer or making the specs file too reliant on knowing how to access `GlobeTranslucencyFramebuffer` properties. I think I've been inconsistent with choosing which getters to add to which classes..

Reply:
Resolved by adding comments above getters that are only used for specs.